crates/tests/mqtt_tests/Cargo.toml

```diff
@@ -11,7 +11,8 @@ rust-version = "1.58.1"
 anyhow = "1.0"
 fastrand = "1.8"
 futures = "0.3"
+maplit = "1"
 once_cell = "1.8"
-rumqttc = "0.17"
-rumqttd = "0.11"
+rumqttc = "0.18.0"
+rumqttd = "0.12.1"
```

Comment:
We agreed not to add the patch number versions.

Reply:
sure, changed it.